File: abc164/a/a.go
package main

import "fmt"

func sheepwolf(s int, w int) bool {
	return s <= w
}

func main() {
	var s, w int
	fmt.Scan(&s, &w)
	if sheepwolf(s, w) {
		fmt.Println("unsafe")
	} else {
		fmt.Println("safe")
	}
}

File: abc164/a/a.ts
import * as fs from "fs";

function sheepwolf(s: number, w: number): boolean {
  return s <= w;
}

function main() {
  const input = fs.readFileSync("/dev/stdin", "utf-8").trim().split(/\s+/).map(Number);
  const s = input[0];
  const w = input[1];

  if (sheepwolf(s, w)) {
    console.log("unsafe");
  }
  else {
    console.log("safe");
  }
}

main();
